app/controllers/events_controller.rb

```diff
@@ -0,0 +1,41 @@
+class EventsController < ApplicationController
+  def index
+    @events = Event.all
```

Comment:
as mentioned above the index page of events should show previous and upcoming events separately